src/sources/splunk_hec.rs

```diff
@@ -284,8 +284,8 @@ struct EventStream<R: Read> {
     events: usize,
     /// Optinal channel from headers
     channel: Option<ValueKind>,
-    /// Extracted default time
-    time: Option<ValueKind>,
+    /// Default time, Ok(extracted), Err(now)
+    time: Result<DateTime<Utc>, DateTime<Utc>>,
```

Comment:
Generally, I don't really like using an error enum in cases where its not really an error. A custom `Time` enum could work here and be a bit more explicit.

Reply:
I agree.